spec/lib/models/node-spec.js

```diff
@@ -152,5 +152,72 @@ describe('Models.Node', function () {
             });
         });
 
+        describe('tags', function() {
+            before(function() {
+                this.subject = this.attributes.tags;
+            });
+
+            it('should be an array', function() {
+                expect(this.subject.type).to.equal('array');
+            });
+
+            it('should default to empty', function() {
+                expect(this.subject.defaultsTo).to.deep.equal([]);
+            });
+        });
+
+        describe('Node Tag Functions', function() {
+            var collection = {
+                update: sinon.stub().resolves()
+            };
+
+            before(function() {
+                sinon.stub(this.model, "native", function(cb) { cb(null, collection); });
```

Comment:
`teachme` question - why stub out `model.native` like this? Not cluing in how it's used by the tests.... (not clear on what `model.native` is - are we just using it to make sure the collections got called and invoked?)